### PR TITLE
feat: add feedback component and API wiring

### PR DESCRIPTION
**Pull request by bgsandan**
Adds customised ngx-feedback module for submitting user feedback

Posted feedback is stored as a GitHub issues - e.g. micronutrientsupport/feedback-imagery#23

### CODE REVIEW

File: src/app/app.component.scss [hunk 1]
```diff
@@ -2,4 +2,9 @@
   overflow: hidden;
   height: 100vh;
   width: 100vw;
-}
\ No newline at end of file
+}
+
+::ng-deep ngx-feedback-main-btn {
```
Comment (Fuhji) [inline src/app/app.component.scss:7]: I just know that Jon would have had something to say about the use of ng-deep..... looks to be deprecated https://blog.angular-university.io/angular-host-context/

**Comment by Fuhji** [comment]
code looks fine (although cypress test failed on first pass), but not sure how can be tested?

**Comment by bgsandan** [comment]
Can always try submitting some 'feedback' :)

**Review by Fuhji — APPROVED** [review]
*borat thumbs up* very nice!

**Pull request merged**